UI/EncyclopediaDetailPanel.cpp

```diff
@@ -2982,18 +2984,15 @@ namespace {
         ScriptingContext context{universe, Empires(), GetGalaxySetupData(), GetSpeciesManager(), GetSupplyManager()};
         universe.InhibitUniverseObjectSignals(true);
 
-        // planet->SetSpecies modifies a string that may be part of the input
-        // string_view vector, so make copies now to use while iterating
-        std::vector<std::string> names_copy(species_names.begin(), species_names.end());
-        for (auto& species_name : names_copy) { // TODO: parallelize somehow? tricky since an existing planet is being modified, rather than adding a test planet...
```

Comment:
if not iterating over `string` but instead `const string_view` then should also remove the `move` on line 3013